### ISSUE TITLE
contrib/jmoiron/sqlx: uint64 argument error

### ISSUE THREAD

**Issue by alexpareto** [DataDog/dd-trace-go#563]
I'm currently setting up the SQLx implementation of Datadog's tracer but am getting this error: `converting argument $1 type: uint64 values with high bit set`.

We're using the mysql driver - and haven't had any issues before implementing the datadog sqlx trace helper. It looks like it's trying to query w/ postgres almost ($1 looks very postgres vs ? in mysql parameterized query syntax).

This only seems to happen in tests. 

any ideas? is dd not using the underlying driver for some reason?

Here is a related issue with mysql implementing the underlying interface to prevent this uint64 error: https://github.com/go-sql-driver/mysql/issues/329

**Comment by cgilmour** [comment]
A snippet that reproduces it:
```go
package main

import (
        "context"
        "log"

        sqltrace "gopkg.in/DataDog/dd-trace-go.v1/contrib/database/sql"
        sqlxtrace "gopkg.in/DataDog/dd-trace-go.v1/contrib/jmoiron/sqlx"

        "github.com/jmoiron/sqlx"
        "github.com/go-sql-driver/mysql"
)

func main() {
        sqltrace.Register("mysql", &mysql.MySQLDriver{})

        connectString := "root:password@tcp(localhost:3306)/"
        db, err := sqlx.Connect("mysql", connectString)
        if err != nil {
                log.Println("unable to connect")
                return
        }
        traced_db, err := sqlxtrace.Connect("mysql", connectString)
        if err != nil {
                log.Println("unable to connect")
                return
        }

        ctx := context.Background()

        var result uint64
        err = db.GetContext(ctx, &result, "SELECT ?", ^uint64(0))
        if err != nil {
                log.Println("error executing raw query:", err)
        }
        err = traced_db.GetContext(ctx, &result, "SELECT ?", ^uint64(0))
        if err != nil {
                log.Println("error executing traced query:", err)
        }
}
```
Running against a mariadb instance in docker
```
docker run --rm -d -e MYSQL_ROOT_PASSWORD=password -p 3306:3306 mariadb:10
```
After it's running, the code above gives this output:
```sh-session
$ go run main.go 
2020/01/09 01:48:14 error executing traced query: sql: converting argument $1 type: uint64 values with high bit set are not supported
$
```

**Comment by gbbr** [comment]
@alexpareto thanks for bringing this to our attentiont and @cgilmour for reproducing. We've acknowledged the issue and will try our best to address it as soon as possible.

In the meantime, if you have ideas, remember that contributions are welcome.

**Comment by gbbr** [comment]
Fixed! We’ll have it out as part of 1.21.0 soon! Thanks for bringing it up!
